pkg/cmd/cluster/bind/bind.go

```diff
@@ -73,7 +73,7 @@ func NewBindCommand(f *factory.Factory) *cobra.Command {
 	return cmd
 }
 
-func runBind(opts *Options) error {
+func runBind(opts *args) error {
```

Comment:
I know there was a discussion around the new name but I don't think we fully agreed on it (myself and @wtrocki coming from different view points). I think the opts variable name could be confusing with an `args` type. On consideration I think it is best to leave it as `options` for now, we can always revisit later but it causes the least disruption.